src/utils/ServerInterfaceStaff.tsx

```diff
@@ -0,0 +1,165 @@
+// Server-client interface for /api/users/staff
+import { staffType } from "./models/staffModel";
+
+const BACKEND_ROUTE = `${import.meta.env.VITE_BACKEND_ROUTE}/users/staff`;
+
+// JSON response checker - TODO: change to use jack's responseHelpers.tsx once merged   
+async function checkJSONResponseStatus(response: Response) {
+    if(!response.ok) {
+        throw new Error(response.statusText);
+    }
+    let parsedResponse = await response.json();
+    return parsedResponse;
+}
+
+// Miscellaneous response checker - TODO: change to use jack's responseHelpers.tsx once merged 
+async function checkMiscResponseStatus(response : Response) {
+    if(!response.ok) {
+        throw new Error(response.statusText);
+    }
+    return true;
+}
```

Comment:
These should be in a separate file since the event and youth routes will need them too. Since Jack has implemented a good version of these methods in his PR, you can take these out.